spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -1,4 +1,5 @@
+
 org.springframework.cloud.bootstrap.BootstrapConfiguration=\
-org.springframework.cloud.gcp.config.autoconfig.GcpConfigAutoConfiguration,\
+org.springframework.cloud.gcp.autoconfigure.config.GcpConfigAutoConfiguration,\
```

Comment:
That's not `BootstrapConfiguration` part. This must be for the `org.springframework.boot.autoconfigure.EnableAutoConfiguration` property

Reply:
I think this is correct, because it's for `BootstrapConfiguration`, as required by Spring Cloud Config.